siemens/ac_pretty.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# parameters: "ac verify report" "ac total"
+ac_report=${1}
+ac_total="$(grep "AC Total" ${ac_report} | tr -s ' ' | cut -d ' ' -f 4)"
+
+echo -----------------------------------------------------------------
+grep "Found" ${ac_report}
+grep "AC Total" ${ac_report}
+echo -----------------------------------------------------------------
+if [[ '0' == ${ac_total} ]]; then
+  echo -e "${GREEN}Autocheck found no new issues${NC}"
```

Comment:
Please match the formatting of the other `GREEN` message by includeing `# ` and ending with `$C`:
` "$(GREEN)# Autocheck found no new issues $C"`
`"${RED}# Autocheck error count: ${ac_total} $C"`

Also, `c` could have a more descriptive name, like `exitcode`. Maybe I'll fix `pretty_run.sh` too.